<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbulEduWidgetSCOLOMFrV1</class>
 <widget class="QWidget" name="AbulEduWidgetSCOLOMFrV1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>598</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>664</width>
    <height>467</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
 border: 1px solid gray;
 border-radius: 5px;
 margin-top: 2ex;
}
 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center;
     padding: 0 3px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="gbScoLOMFRRequires">
     <property name="title">
      <string>Indexation ScoLOMFR</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>901</width>
           <height>406</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <layout class="QGridLayout" name="glLangues" rowstretch="0,0">
              <property name="horizontalSpacing">
               <number>10</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="lblSCOLOMFR1_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>1.3 Langue </string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="pushBbtnPlusSCOLOMFR1_3utton">
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="cbScoLOMFR1_3"/>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="glDescription">
            <property name="spacing">
             <number>4</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lblSCOLOMFR1_4">
              <property name="text">
               <string>1.4 Description </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cbSCOLOMFR1_4Langs"/>
            </item>
            <item row="0" column="2">
             <widget class="QTextEdit" name="teSCOLOMFR1_4">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>60</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="btnPlusSCOLOMFR1_4">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QVBoxLayout" name="vlContributions">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="lblSCOLOMFR2_3">
              <property name="text">
               <string>2.3 Contributions</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="AbulEduWidgetContribution" name="frScoLOMFR2_3">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="glFrScoLOMFR2_3">
               <property name="margin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>4</number>
               </property>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnPlus2_3">
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <layout class="QGridLayout" name="glLocalisation">
              <property name="horizontalSpacing">
               <number>4</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="lblScoLOMFR4_3">
                <property name="text">
                 <string>4.3 Localisation</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="leScoLOMFR4_3">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="btnPlusScoLOMFR4_3">
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblLogoScolomFR">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="AbulEduWidgetLOM.qrc">:/abuleduwidgetlom/icones/logo_scolomfr.gif</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblRaconteMoiSCOLOMFR">
     <property name="text">
      <string>Ce Logiciel respecte également la norme ScoLOMFR.
La saisie des informations ci-dessus permet l'indexation de ce module dans le respect de la norme.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frAuthenticate">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblAuthenticate">
        <property name="text">
         <string>Vous devez être authentifié pour modifier ou compléter ces informations</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="hsAutheticate">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>260</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnAuthenticate">
        <property name="text">
         <string>S'authentifier</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AbulEduWidgetContribution</class>
   <extends>QFrame</extends>
   <header>abuleduwidgetcontribution.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>cbScoLOMFR1_3</tabstop>
  <tabstop>pushBbtnPlusSCOLOMFR1_3utton</tabstop>
  <tabstop>cbSCOLOMFR1_4Langs</tabstop>
  <tabstop>teSCOLOMFR1_4</tabstop>
  <tabstop>btnPlusSCOLOMFR1_4</tabstop>
  <tabstop>btnPlus2_3</tabstop>
  <tabstop>leScoLOMFR4_3</tabstop>
  <tabstop>btnPlusScoLOMFR4_3</tabstop>
  <tabstop>btnAuthenticate</tabstop>
 </tabstops>
 <resources>
  <include location="AbulEduWidgetLOM.qrc"/>
 </resources>
 <connections/>
</ui>
